security/account-password-policy.yaml

```diff
@@ -141,24 +141,24 @@ Resources:
               const done = (err) => {
                 if (err) {
                   console.log(`Error: ${!JSON.stringify(err)}`);
-                  response.send(event, context, response.FAILED, {});
+                  response.send(event, context, response.FAILED, {}, context.log_stream_name);
                 } else {
-                  response.send(event, context, response.SUCCESS, {});
+                  response.send(event, context, response.SUCCESS, {}, context.log_stream_name);
```

Comment:
can you revert the `, context.log_stream_name`
this was introduced in the 2nd commit and is the default